roles/uperf/templates/workload_vm.yml.j2

```diff
@@ -90,7 +90,7 @@ spec:
           - bash /tmp/uperf_script/run_script.sh
     name: cloudinitdisk
   - configMap:
-      name: uperf-test-{{ trunc_uuid }}
+      name: uperf-test-{{ item }}-{{ trunc_uuid }}
```

Comment:
@smalleni, I think this should be `uperf-test-{{ ansible_loop.index0 }}-{{ trunc_uuid }}`